src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/WebRequestSession.cs

```diff
@@ -105,8 +158,134 @@ public WebRequestSession()
         internal void AddCertificate(X509Certificate certificate)
         {
             Certificates ??= new X509CertificateCollection();
+            if (!Certificates.Contains(certificate))
+            {
+                ResetClient();
+                Certificates.Add(certificate);
+            }
+        }
+
+        internal virtual HttpClient GetHttpClient(bool handleRedirect, out bool clientWasReset)
```

Comment:
Why virtual?

Reply:
Accidentally copied from the original location. Removed.